src/main/cpp/commands/SetConveyor.cpp

```diff
@@ -0,0 +1,30 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+#include "commands/SetConveyor.h"
+
+SetConveyor::SetConveyor(Conveyor& conveyor) 
+: mConveyor{conveyor}
+{
+  // Use addRequirements() here to declare subsystem dependencies.
+  AddRequirements(&mConveyor);
+}
+
+// Called when the command is initially scheduled.
+void SetConveyor::Initialize() {}
+
+// Called repeatedly when this Command is scheduled to run
+void SetConveyor::Execute() 
+{
+  mConveyor.SetConveyor();
```

Comment:
The code in `Conveyor::SetConveyor` should be here. `Conveyor::SetConveyor` should be much more simple.